src/workflows/runCRISPRCasFinderl_array.cwl

```diff
@@ -0,0 +1,60 @@
+cwlVersion: v1.2
+class: Workflow
+label: runCRISPRCasFinder
+doc: |
+      Implementation of CRISPRCasFinder processing for multiple input fasta files.
+
+requirements:
+  SubworkflowFeatureRequirement: {}
+  ResourceRequirement:
+    coresMin: 1
+    ramMin: 2000
+  ScatterFeatureRequirement: {}
+
+inputs:
+  sequences:
+    type: File[]
+    format: edam:format_1929
+    label: input Fasta file to analyze
+  soFile:
+    type: string?
+    label: path to the sel392v2.so file, required by vmatch
+    default: /opt/CRISPRCasFinder/sel392v2.so
+  casDefinition:
+    type: string?
+    label: Cas-finder definition, such as G (general), T (Typing) or S (Subtyping)
+    default: G
+
+outputs:
+  crisprcasfinder_fasta:
+    type: File[]
+    format: edam:format_1929
+    outputSource: full_wf/crisprcasfinder_fasta
+  crisprcasfinder_gff:
+    type: File[]
+    format: edam:format_1975
+    outputSource: full_wf/crisprcasfinder_gff
+
+steps:
+    full_wf:
+      label: run full CRIPRCasFinder workflow for multiple files
+      run: runCRISPRCasFinderl.cwl
+      scatter: sequences
+      in:
+        sequences: sequences
+        soFile: soFile
+        casDefinition: casDefinition
```

Comment:
We should try to keep the naming consistent:
`casDefinition` and then the outputs are `crisprcasfinder_fasta`
I would go with `cas_definition`